Scripts/common.psm1

```diff
@@ -38,19 +38,21 @@ function Invoke-DotnetBuild([String]$dotnet, [String]$solution, [String]$config,
     Write-Comment -prefix "..." -text "Building $solution"
 
     $platform = "/p:Platform=`"Any CPU`""
-    $restore_command = "restore $solution"
-    $build_command = "build -c $config $solution --no-restore"
+    $restore_command = "restore $solution $platform"
```

Comment:
nice code reuse catch here!

Reply:
yeah, thanks, it wasn't working for me without this change, and looks like it still works in the [cloud build ](https://github.com/microsoft/coyote/actions/runs/3327994415) also.